package.json

```
@@ -28,13 +28,14 @@
   "scripts": {
     "build": "ts-node ./script/build.ts && tsc --project tsconfig.build.json",
     "build:tokens": "node -e \"require('./build')._init()\"",
-    "format": "prettier --check '**/*.{js,jsx,ts,tsx,md,mdx,css,json}'",
+    "format": "prettier --check '**/*.{js,jsx,ts,tsx,md,mdx,css}'",
     "lint": "eslint '**/*.{js,ts,tsx,md,mdx}' --max-warnings=0 --config .eslintrc.js",
     "postbuild": "ts-node ./script/color-contrast.ts",
     "prebuild": "rm -rf dist",
     "prebuild:tokens": "rm -rf tokens-v2-private",
-    "prepack": "yarn build && yarn build:tokens",
+    "prepack": "yarn build && yarn build:tokens && yarn tempColorTokens",
     "release": "changeset publish",
+    "tempColorTokens": "ts-node tempColorTokenBuild.ts",
```

sorry, missed this one before.. one final request for consistency..

```suggestion
    "build:color-tokens": "ts-node tempColorTokenBuild.ts",
```

---

also, can we move this file to `scripts/` folder? Only reason `build.js` isn't, is because it's surfaced in the npm package.

---

We can do this for now. In the end this will be integrated / updating `build.ts`.